### PR TITLE
Merge #13622: Remove mapRequest tracking that just effects Qt display.

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Fixes CVE-2018-17145 (https://invdos.net/paper/CVE-2018-17145.pdf) 

Bug was introduced in bitcoin#10286, which we merged in https://github.com/dashpay/dash/pull/3353

DoSe vector only exists in 0.16.0rc1, rc2. Does not exist in 0.15. Thanks to xkcd on discord for bring it to my attention

### CODE REVIEW

File: src/qt/transactiontablemodel.cpp [hunk 1]
```diff
@@ -520,8 +514,6 @@ QVariant TransactionTableModel::txStatusDecoration(const TransactionRecord *wtx)
     case TransactionStatus::OpenUntilBlock:
     case TransactionStatus::OpenUntilDate:
         return GUIUtil::getThemedQColor(GUIUtil::ThemedColor::TX_STATUS_OPENUNTILDATE);
-    case TransactionStatus::Offline:
```
Comment (UdjinM6) [inline src/qt/transactiontablemodel.cpp:523]: Should drop `TX_STATUS_OFFLINE`

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/qt/transactiontablemodel.cpp (hunk 1 above)
Comment (PastaPastaPasta) [reply src/qt/transactiontablemodel.cpp:523]: https://github.com/dashpay/dash/pull/3694/commits/c506b1113e0127f26748fbfd7b721170032019e0

Comment (UdjinM6) [reply src/qt/transactiontablemodel.cpp:523]: c506b11 isn't enough, should also drop https://github.com/dashpay/dash/blob/develop/src/qt/guiutil.cpp#L156 and https://github.com/dashpay/dash/blob/develop/src/qt/guiutil.cpp#L172

Comment (PastaPastaPasta) [reply src/qt/transactiontablemodel.cpp:523]: git grep is useful... I should use it. https://github.com/dashpay/dash/pull/3694/commits/d8f8592077ab05abdbfeb8f4a2f61b66bcbc6f0d

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by xdustinface — APPROVED** [review]
utACK

**Pull request merged**